VarduhiGhukasyan/Hash Table

```diff
@@ -0,0 +1,92 @@
+#include <iostream>
+#include <functional>
+#include <vector>
+#include <list>
+
+std::hash<int> myhash;
+
+class Hash
+{
+public:
+    Hash(int _size)
+        : table_size(_size)
+    {
+        hash_table.resize(table_size);
+    }
+
+    void insert(int x)
+    {
+        int index = myhash(x) % table_size;
+        hash_table[index].push_back(x);
+    }
+
+    void deleteElem(int x)
+    {
+        int index = myhash(x) % table_size;
+        auto IT = hash_table[index].begin();
+        for (; IT != hash_table[index].end(); ++IT)
+        {
+            if (*IT == x)
+            {
+                break;
+            }
+        }
+        if (IT != hash_table[index].end()) hash_table[index].erase(IT);
```

Comment:
The only case missing if there are multiple occupancies of the value you are trying to remove, is that expected functionality?